ansible/group_vars/all/vars.yml

```diff
@@ -0,0 +1,55 @@
+---
+passwd:
+  # services
+  afredis:  {login: afredis,  id: 2000, comment: Airflow Redis}
+  afpsql:   {login: afpsql,   id: 2001, comment: Airflow Postgresql}
+  afsched:  {login: afsched,  id: 2002, comment: Airflow Scheduler}
+  afwebsrv: {login: afwebsrv, id: 2003, comment: Airflow Webserver}
+  afworker: {login: afworker, id: 2004, comment: Airflow Worker}
+  afflower: {login: afflower, id: 2005, comment: Airflow Celery Flower}
+  plpsql:   {login: plpsql,   id: 2100, comment: Pipeline Postgresql}
+  plshovel: {login: plshovel, id: 2101, comment: Pipeline Shovel}
+  # people
+  agrabeli:
+    login: agrabeli
+    id: 10000
+    comment: Maria Xynou
+    keys: ["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDD0JSwM+t3Uz9lS3Mjoz9oo4vOToWyzboZhYQbP8JY5HvFtAvWanWHnUBO91t6hkgKIMiUqhdCJn26fqkhSGe/bRBaFUocOmuyfcmZoRdi0qzAskmycJsj/w6vWR4x6MYkmJvSeI/MGxjEFt4s2MfOG1tP8CBLUYft9qUleeJa7Jln8c+xbnqB7YngaI190icQHE9NuIB2CXvzbmo3tLtHNMagEwI7VoBDj6mxzTxBd9JhuhF4w5uGxxm0Gp1hzk+15obNnaBS+Anr7jXz8FPwwxCH+XhBZxB1PPpcIayKrf9iLyGtwmhkdDoWCqYAr1mue3LxFso+TZF4bwE4Cjt1 agrabelh@agrabelh"]
+  weasel:
+    login: "{{ vault_weasel_login }}"
+    id: 10001
+    comment: "{{ vault_weasel_comment }}"
+    keys: "{{ vault_weasel_keys }}"
+  art:
+    login: art
+    id: 10002
+    comment: Arturo Filasto
+    keys: ["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC9her6lZIfP/ulXleu00u3ukdcVEDslkfXzi65JFRXRkKYsdMw7zLvrkvFryqDGsRQwpZo0rlk5o26QNSUR/g2Kf3wEHxxmMcN4+g+f0/fJDKLHSFdL5Z5ACyp5YvjA88vG753qVn7lBINuvCb4wk1sUOWM1WVG5WiszLiJCddYUrgCZWzD3FHyApJ/GGH2A5vTTN8mcQapQaJ9/sgxDw5WT+yMtZ8TW4462x+TaWWGE0WnAd/l6HOFaS14mql0K+MbtRp9EKN2O6+sA0OsjknYpt21ovtEWfBiojjh+4Iz6bTk+M5+fWHwQaxqEcaok22l35GudlmaoTB0OesB/1z art@eclipsis"]
+  darkk:
+    login: darkk
+    id: 10003
+    comment: Leonid Evdokimov
+    keys: ["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDWe9jGACZhSKhCq/jGOe8xKpY8bPSC8MagEiWMCzcHA5520Js4EGzwuYx5h/7sHK4GKWekGzO57xOsgeKACDKo8JppIPt+uJtnwJpVr7EHdnX1hhBMfo0NfV42R+//fZIHMNtGIW31iPg3cQVp9c+GQPsdmuDPbKGLoQvS9dGnF+N6lM+dicEmeBrqbA/t5V1j/rrfpmpTNOWef2WsREFdZBQbgD3rZU1lG0CnxNw6ATnFQgqcC1QsyfWD8e1pmDLByoVuYJ6WXBLKgI86hrMNidxGjzmpAn2q+5cbu5HZXOkmny3TBiP0VxPJJ43PuLIFkdCNh50KRPtTug3o38Oj leon+2016@darkk.net.ru"]
+  nuke:
+    login: nuke
+    id: 10004
+    comment: Lorenzo Primiterra
+    keys: ["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC+QtDwA/s8vw7aETWGjItUBmgCXgm9jTEjCh/5OofGJ/Bk4kjZ+a6NisUNJRvrySfr89oeJz7VOuHgHTCM1g2CKANAiidcBkepHjdBlHDElV1WBlUxdgl0RsbM4ue8zUxtrmzEQaxLxL769ZHR+gI8PoZau1cSyKG5jVge91Bt1+wvV3dMGvxibM144+uHCrxHpBsbRd9irIgk+vNJmecKn3bSh3aKo9txBjIqBprBrZ1Dqj0x9IKVhD59URcO1Vrr3HXPghZnsWFjOSSQ0DeVpkE3cEz78CpdvqL0drv4Mo1ZgE33YLwWjJSceDGIrtnsPZhs5YzOI4iStRIdPD4X nuke"]
```

Comment:
I'm not sure how this variable gets used, but the [user](http://docs.ansible.com/ansible/user_module.html) module lets you set `system: yes` on accounts.  I would suggest using this, and letting the system automatically pick random UID's in the system range for the `prometheus` user.